INSTALL.md

```diff
@@ -34,22 +36,13 @@ Python Prerequisites
 - pandas
 - psutil
 - pyproj
+- pyyaml
```

Comment:
should add: six
we also install pyuvdata and h5py in our travis setup, not sure if those are needed as well.

Reply:
interestingly, we don't install pyyaml in Travis...